packages/ag-charts-community/src/chart/dom/domStyles.css

```diff
@@ -1,14 +1,67 @@
 /********
 * RESET *
 ********/
-[class^='ag-charts'],
-[class^='ag-charts']:focus,
-[class^='ag-charts']:after,
-[class^='ag-charts']:before {
+.ag-charts-root,
+.ag-charts-root:focus,
+.ag-charts-root:after,
+.ag-charts-root:before,
+.ag-charts-root *,
+.ag-charts-root *:focus,
+.ag-charts-root *:after,
+.ag-charts-root *:before {
```

Comment:
These have the same specificity, but this way is a lot better for performance. Modern CSS engines have things like bloom filters for class names so avoids a lot of substring checks

Reply:
Presuming that all of the focus boxes still appear as before?